client/blocks/upload-image/index.jsx

```diff
@@ -246,6 +253,19 @@ class UploadImage extends Component {
 		this.setState( { uploadedImage: null } );
 	};
 
+	componentWillMount() {
+		// use defaultImage as uploadedImage if set.
+		const { defaultImage } = this.props;
+		if ( defaultImage ) {
+			this.setState( {
+				uploadedImage: defaultImage,
+				selectedImage: defaultImage.URL,
```

Comment:
That's why the prop is prefixed with `default` which should be hint to developers that it shouldn't be set willy nilly. When there is a need to support dynamically settable image, we can add a new prop named `image` although that will conflict with class name. Ultimately, we should just add a new class named `EditableImage` or something.